packages/pixeloven-cli-addon-webpack/src/commands/webpack.test.ts

```diff
@@ -1,23 +1,24 @@
-import "jest";
-import sinon from "sinon";
+import { cli, Sandbox } from "../testing";
 import webpackModule from "./webpack";
 
-const sandbox = sinon.createSandbox();
-
 describe("@pixeloven/cli-addon-webpack", () => {
     describe("commands", () => {
         describe("webpack", () => {
             afterEach(() => {
-                sandbox.reset();
+                Sandbox.reset();
             });
             afterAll(() => {
-                sandbox.restore();
+                Sandbox.restore();
             });
             it("should contains required props", () => {
                 expect(webpackModule.alias).toEqual(["--webpack", "-w"]);
                 expect(webpackModule.name).toEqual("webpack");
                 expect(typeof webpackModule.run).toEqual("function");
             });
+            xit("should throw an error without run type", async () => {
```

Comment:
This one won't work until I can figure out how to test addons like I do with the main CLI.